recipes/qtawesome/all/test_package/test_package.cpp

```diff
@@ -1,4 +1,4 @@
-#include "QtAwesome.h"
+#include "QtAwesome/QtAwesome.h"
```

Comment:
How has this header changed? Is this compatible with both `qtawesome/6.4.0` and `qtawesome/6.7.2+p1`

Reply:
Should we add this (below) to the `package_info` instead of changing that header?

```python
        self.cpp_info.includedirs = [os.path.join(self.package_folder, "include", "QtAwesome")]
```